validator/tools/validate_relationship_rules.js

```diff
@@ -0,0 +1,133 @@
+const utils = require('./utils')
```

Comment:
This is the same as `validate_rules.js` and not needed right?

Reply:
oops! thanks :D